uaclient/gpg.py

```diff
@@ -1,55 +1,24 @@
 import logging
 import os
+import shutil
 
 from uaclient import exceptions
-from uaclient import util
 
 
-def export_gpg_key_from_keyring(
-    key_id: str, source_keyring_file: str, destination_keyfile: str
-) -> None:
-    """Export a specific key from source_keyring_file into destination_keyfile
+def export_gpg_key(source_keyfile: str, destination_keyfile: str) -> None:
```

Comment:
Something of a nit, but I wonder if this would be better off just taking a single `str` argument and handling constructing the source and destination paths itself? It would push a little complexity down the stack, making the already-more-complex entitlement class a little simpler.